login/scss/detail.scss

```diff
@@ -0,0 +1,66 @@
+body {  
```

Comment:
Don't style the `body` directly. If you need a new container, create a wrapper `div`